packages/autometrics/src/exporter-prometheus/mod.ts

```diff
@@ -0,0 +1,66 @@
+import { MetricReader } from "npm:@opentelemetry/sdk-metrics@^1.17.0";
+
+import {
+  BuildInfo,
+  amLogger,
+  createDefaultBuildInfo,
+  recordBuildInfo,
+  registerExporter,
+} from "../../mod.ts";
+import { PrometheusExporter } from "./PrometheusExporter.ts";
+
+let metricReader: MetricReader | undefined;
+
+export type InitOptions = {
+  /**
+   * Optional build info to be added to the `build_info` metric.
+   */
+  buildInfo?: BuildInfo;
+
+  /**
+   * Hostname or IP address on which to listen.
+   *
+   * @default '0.0.0.0'
+   */
+  hostname?: string;
+
+  /**
+   * Port on which to open the Prometheus scrape endpoint (default: 9464).
+   */
+  port?: number;
+};
+
+/**
+ * Initializes and registers the Prometheus exporter for Autometrics.
+ *
+ * This opens up a webserver with the `/metrics` endpoint, to be scraped by
+ * Prometheus.
+ */
+export function init({
+  buildInfo,
+  hostname = "0.0.0.0",
+  port = 9464,
+}: InitOptions = {}) {
+  if (metricReader) {
+    throw new Error("Prometheus exporter is already running");
+  }
+
+  amLogger.info(`Opening a Prometheus scrape endpoint at port ${port}`);
+
+  metricReader = new PrometheusExporter({ host: hostname, port });
+  registerExporter({ metricReader });
+
+  recordBuildInfo(buildInfo ?? createDefaultBuildInfo());
+}
+
+/**
+ * Stops the built-in Prometheus exporter.
+ */
+export async function stop() {
```

Comment:
What is the purpose for the `stop()` interface? I understand it's made available for the library user

Reply:
It's primarily for the tests, but it's more that I didn't see a good reason to not expose it since we have it anyway :)